### PR TITLE
Design

### PR DESCRIPTION
**Pull request by taltaub22**
alt text

alt text

### CODE REVIEW

**Comment by idanstark42** [comment]
Great usage of the UI package!
I am missing some stuff though:
 - You're not using any icons, which is very important in UX, and we have the entire free, fontawsome library in our UI. Inline buttons especcially, should have the `clear` class and contain only an icon
 - The `active` class for the router links doesn't seem to work, the "is-active" class is not added to the current page's link, and same goes for the settings
 - Credit page doesn't exist...
 - If you add to the `div` that comes after the `top-bar` a class called `top-par-page` it will be scrollable, leaving the `top-bar` to be always visible
 - Why is no option selected when I go into the `settings` page?
 - Don't write "loading", there is a loader in the library:
Add this HTML inside the element you want to load:
```html
<div class="dimmer">
    <div class="loader"></div>
</div>
```
Then, when you add the `loading` class to the parent that contains this HTML, you will see the loader fill the element, and nothing is clickable. Also you can add options to the loader by adding on top of the `loader` class the following classes: slow, slower, slowest, fast, faster, fastest, extreme, counter-clockwise, tiny, small, big, large, huge

File: appveyor.yml [hunk 1]
```diff
@@ -20,8 +20,7 @@ install:
 build_script:
   - yarn run build:win --c.directories.output=./dist/
   - >-
-    "C:\\Program Files (x86)\\Inno Setup 5\\ISCC.exe" /Odist
-    /FFIRST_LEGO_League_Tournament_Management_System_Setup_%APPVEYOR_BUILD_VERSION% .\\windows\\setup\\main.iss
+    "C:\\Program Files (x86)\\Inno Setup 5\\ISCC.exe" /Odist /FFIRST_LEGO_League_Tournament_Management_System_Setup_%APPVEYOR_BUILD_VERSION% .\\windows\\setup\\main.iss
```
Comment (2roy999) [inline appveyor.yml]: don't

File: src/renderer/components/home.vue [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
+        <h1>Tournament Management System</h1>
+
         <table>
             <tbody>
             <tr v-for="module in modules">
-                <td>{{module.name}}:</td>
+                <td>{{module.name}}</td>
                 <td><a @click="event => openSite(event, module.site)" :href="module.site">{{module.site}}</a></td>
-                <td><button @click="() => saveInClipboard(module.site)">Copy</button></td>
+                <td><button class="button" @click="() => saveInClipboard(module.site)">click</button></td>
```
Comment (2roy999) [inline src/renderer/components/home.vue]: click?

File: package.json [hunk 3]
```diff
@@ -34,6 +34,7 @@
     ]
   },
   "dependencies": {
+    "@first-lego-league/user-interface": "^1.2.10",
```
Comment (2roy999) [inline package.json]: move to dev-dependencies and use webpack to bring the css

File: src/index.ejs [hunk 4]
```diff
@@ -9,6 +9,9 @@
         require('module').globalPaths.push('<%= htmlWebpackPlugin.options.nodeModules.replace(/\\/g, '\\\\') %>')
       </script>
     <% } %>
+
+      <link rel="stylesheet" type="text/css" href="../node_modules/@first-lego-league/user-interface/current/assets/css/app.css">
```
Comment (2roy999) [inline src/index.ejs]: Use webpack to bring the css

File: src/index.ejs [hunk 5]
```diff
@@ -9,6 +9,9 @@
...
+      <script src="../node_modules/@first-lego-league/user-interface/current/assets/js/app.js"></script>
```
Comment (2roy999) [inline src/index.ejs]: Don't add a external js file for the ui (you don't need it)

File: appveyor.yml (hunk 1 above)
Comment (taltaub22) [reply appveyor.yml]: what?

**Comment by taltaub22** [comment]
I think its time to merge this

File: src/main/mhub/index.js [hunk 6]
```diff
@@ -22,7 +22,7 @@ const MHUB_EXECUTABLE_PATH = path.resolve('./internals/mhub/bin/mhub-server')
 const MHUB_FILE_TEMPLATE = path.join(__static, 'mhub-config.ejs')
 const MHUB_FILE_PATH = path.resolve('./tmp/$mhub.config.json')
 
-const MAX_RETRIES = 5
+const MAX_RETRIES = 15
```
Comment (2roy999) [inline src/main/mhub/index.js]: I fixed the problem here you can return it to 5

File: appveyor.yml [hunk 7]
```diff
@@ -1,8 +1,3 @@
-branches:
```
Comment (2roy999) [inline appveyor.yml]: Please return this part

**Pull request merged**